packages/lib/src/spatial-navigation/components/virtualizedList/helpers/getNumberOfItemsVisibleOnScreen.ts

```diff
@@ -0,0 +1,43 @@
+const getMinSizeOfItems = <T>({
+  data,
+  itemSize,
+}: {
+  data: T[];
+  itemSize: number | ((item: T) => number);
+}) => {
+  if (typeof itemSize === 'number') {
+    return itemSize;
+  }
+
+  let minSize: number | undefined = undefined;
+
+  for (let i = 0; i < data.length; i++) {
+    const item = data[i];
+    if (item !== undefined) {
+      const currentSize = itemSize(item);
+      if (minSize === undefined || currentSize < minSize) {
```

Comment:
Shouldn't this be called `getMaxSize` given that the check is `currentSize < minSize  =>  minSize = currentSize` ?

Reply:
Maybe the naming or the code is not clear, but the value I wanted to get here was the size of the smallest item in the list (hence the "min" size) in order to cover for the "worst case scenario" where we have only small items displayed.
Maybe a better naming could be something like `getSafeItemSize` ?
I'm very open to suggestions here haha

Reply:
My by, i got confused with the condition 😅
I guess this could be done with `.reduce()` but i'm not sure it would make it clearer

Reply:
I also used a reduce in the last commit, it is indeed clearer :)